_example/example.tf

```diff
@@ -13,10 +22,11 @@ module "service-account" {
   label_order = var.label_order
 
   service_account_enabled     = true
-  project_id                  = "clouddrove"
+  project_id                  = "clouddrove-1"
   service_account_key_enabled = true
```

Comment:
I think this resource also needed as bool
**google_service_account_iam_binding**  which mentioned in root main.tf

Reply:
- done sir